tests/testthat/test-summarize_weighted_production.R

```diff
@@ -186,7 +186,17 @@ test_that("preserves groups passed to ...", {
 })
 
 test_that("with demo data returns known value", {
-  master <- prioritize(match_name(loanbook_demo, ald_demo)) %>%
+  allows_reserved_columns <- exists(
+    'allow_reserved_columns',
+    where = asNamespace('r2dii.match'),
+    mode = 'function'
+  )
+  skip_if_not(allows_reserved_columns)
```

Comment:
@jdhoffa this is ready for your review.

Reply:
I'm still getting plenty of test failures when I run the tests locally. This is with the dev version of `r2dii.*`

Reply:
Thanks! Good that I now see the failures on gh-actions. I'll check them out.